espnet/nets/e2e_asr_th.py

```diff
@@ -379,6 +380,9 @@ def set_forget_bias_to_one(bias):
         # https://discuss.pytorch.org/t/set-forget-gate-bias-of-lstm/1745
         for l in six.moves.range(len(self.dec.decoder)):
             set_forget_bias_to_one(self.dec.decoder[l].bias_ih)
+        for n, p in self.named_parameters():
```

Comment:
We initialize this way intendedly as chainer's default initialization (NStepLSTM does not init forget bias to one). Have you observed any improvement by using your new initialization here?

Reply:
That is the reason why your `pytest -k test_lecun_init_torch` failed in Travis-CI. Actually, I found that setting forget bias to 1 does not improve the ASR result at this time https://github.com/espnet/espnet/issues/9 (not written here but I remember)

Reply:
@ShigekiKarita I just did the same manner in the decoder network.
I didn't see any performance differences, so I'll remove this part.